apps/frontpage/_redirects

```diff
@@ -0,0 +1,9 @@
+/addons                https://storybook-addon-catalog.netlify.app/addons                                                     301
+/addons/*              https://storybook-addon-catalog.netlify.app/addons/:splat                                              301
+/blog/*                https://storybook-blog.netlify.app/blog/:splat                                                         301
+/day/*                 https://storybook-day-2023.netlify.app/day/:splat                                                      301
+/showcase/*            https://storybook-component-catalog.netlify.app/showcase/:splat                                        301
+/tutorials/*           https://storybook-tutorials.netlify.app/tutorials/:splat                                               301
+/event-log             https://us-central1-storybook-warehouse.cloudfunctions.net/storybook-event-log-production-event-log    301
+/whats-new/v1/*        https://storybook-dx.netlify.app/.netlify/functions/whats-new/:splat                                   301
+/proxy/api/event       https://plausible.io/api/event                                                                         301
```

Comment:
> When you assign an [HTTP status code](https://docs.netlify.com/routing/redirects/redirect-options/#http-status-codes) of 200 to a redirect rule, it becomes a rewrite. This means that the URL in the visitor’s address bar remains the same, while Netlify’s servers fetch the new location behind the scenes.
— https://docs.netlify.com/routing/redirects/rewrites-proxies/

```suggestion
/addons                https://storybook-addon-catalog.netlify.app/addons                                                     200
/addons/*              https://storybook-addon-catalog.netlify.app/addons/:splat                                              200
/blog/*                https://storybook-blog.netlify.app/blog/:splat                                                         200
/day/*                 https://storybook-day-2023.netlify.app/day/:splat                                                      200
/showcase/*            https://storybook-component-catalog.netlify.app/showcase/:splat                                        200
/tutorials/*           https://storybook-tutorials.netlify.app/tutorials/:splat                                               200
/event-log             https://us-central1-storybook-warehouse.cloudfunctions.net/storybook-event-log-production-event-log    200
/whats-new/v1/*        https://storybook-dx.netlify.app/.netlify/functions/whats-new/:splat                                   200
/proxy/api/event       https://plausible.io/api/event                                                                         200
```

Reply:
Thanks a lot @kylegach. I found another one of their page showing example with 301 so I was confused. I'll try your change.